pandas/core/generic.py

```diff
@@ -1795,7 +1794,7 @@ def empty(self) -> bool_t:
     def __array__(self, dtype=None) -> np.ndarray:
         return np.asarray(self._values, dtype=dtype)
 
-    def __array_wrap__(self, result, context=None):
+    def __array_wrap__(self, result):
```

Comment:
This might (potentially, I don't know) still be passed by numpy, even if we don't use it in the implementation. 
See eg https://docs.scipy.org/doc/numpy/reference/arrays.classes.html#numpy.class.__array_wrap__